checker/client.go

```diff
@@ -58,7 +58,7 @@ func GetClients(ctx context.Context, repoURI, localURI string, logger *log.Logge
 	var repoClient clients.RepoClient
 
 	//nolint:nestif
-	if experimental && glrepo.DetectGitLab(repoURI) {
```

Comment:
There are reasons to use `SCORECARD_EXPERIMENTAL` for GitHub repos as well. This change forces the gitlab repo client.

Reply:
Marking unresolved, as this still looks like it's an issue. Try running this to see my point:
```
SCORECARD_EXPERIMENTAL=1 go run main.go --repo https://github.com/ossf/scorecard
```